backend/getData.js

```diff
@@ -127,12 +127,17 @@ function getData(internalInstance: Object): DataType {
 
   if (internalInstance._instance) {
     var inst = internalInstance._instance;
+
+    // A forceUpdate for stateless (functional) components.
+    var forceUpdate = inst.forceUpdate || function(cb) {
+      inst.updater.enqueueForceUpdate(this, cb, 'forceUpdate');
```

Comment:
very cool. could you make it `inst.forceUpdate || (inst.updater.enqueueForceUpdate && function(cb) {....`?